x/harvest/keeper/deposit.go

```diff
@@ -125,12 +96,8 @@ func (k Keeper) Withdraw(ctx sdk.Context, depositor sdk.AccAddress, amount sdk.C
 }
 
 // GetTotalDeposited returns the total amount deposited for the input deposit type and deposit denom
-func (k Keeper) GetTotalDeposited(ctx sdk.Context, depositType types.DepositType, depositDenom string) (total sdk.Int) {
-
+func (k Keeper) GetTotalDeposited(ctx sdk.Context, depositDenom string) (total sdk.Int) {
```

Comment:
This will have to be tracked separately from the module account once borrows are enabled.

Reply:
See https://github.com/Kava-Labs/kava/pull/716#discussion_r522122035